com.unity.render-pipelines.core/Runtime/Volume/VolumeComponent.cs

```diff
@@ -81,7 +81,9 @@ public class VolumeComponent : ScriptableObject
         /// <summary>
         /// Extracts all the <see cref="VolumeParameter"/>s defined in this class and nested classes.
         /// </summary>
-        static void GetParameters(object o, List<VolumeParameter> parameters)
+        /// <param name="parameters">The list filled with the parameters.</param>
+        /// <param name="filter">If you want to filter the parameters</param>
+        public static void FindParameters(object o, List<VolumeParameter> parameters, Func<FieldInfo, bool> filter = null)
```

Comment:
missing documentation on `o`

Public API must be fully documented in code (even returns if not void). We have test on the API documentation actually. 

If the visibility change is only to have access in VolumeComponent, normaly Editor package see internals on Runtime version of the package usually in Unity. So it would have worked with internal normally.

(optional) Also it do not search, but deliver them up to an optional filter. Get have my preference in this case.

Reply:
Made internal and fix in the documentation, thanks.